CMakeLists.txt

```
@@ -41,6 +37,17 @@ option(OTIO_AUTOMATIC_SUBMODULES "Fetch submodules automatically" ON)
 # Set option dependent variables
 
 if(OTIO_PYTHON_INSTALL)
+    # Find the python interpreter using FindPython introduced in CMake 3.12.
+    # The Development.Module component is used instead of the Development one
+    # because we don't need the libpython library since we don't link against it.
+    # pybind11 will detect that we already found a python
+    # interpreter and will use what was found previously (here).
+    # See https://pybind11.readthedocs.io/en/stable/compiling.html#findpython-mode
+    # Also, the "manylinux" docker images don't have the libpythons anyway.
```

```suggestion
    # Also, the "manylinux" docker images don't have libpython, so requiring the libraries is an issue there.
```
This suggestion is so that in the future, people don't try to turn it back on.

---

Perfect, will change that!